Build failure on Hyprland git

Build fails with:
```
[1/2] Compiling C++ object libhyprsplit.so.p/src_main.cpp.o
FAILED: libhyprsplit.so.p/src_main.cpp.o 
g++ -Ilibhyprsplit.so.p -I. -I.. -I/nix/store/z3bwcdgysdazs43nbj7ilgn6z0s83np6-hyprland-0.49>
../src/main.cpp: In function 'SDispatchResult swapActiveWorkspaces(std::string)':
../src/main.cpp:209:22: error: 'class CWorkspace' has no member named 'moveToMonitor'
  209 |         PWORKSPACEA->moveToMonitor(PMON2->m_id);
      |                      ^~~~~~~~~~~~~
../src/main.cpp:234:22: error: 'class CWorkspace' has no member named 'moveToMonitor'
  234 |         PWORKSPACEB->moveToMonitor(PMON1->m_id);
      |                      ^~~~~~~~~~~~~
ninja: build stopped: subcommand failed.
```

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -206,7 +206,7 @@ SDispatchResult swapActiveWorkspaces(std::string args) {
     if (LAYOUTNAME == "dwindle" || LAYOUTNAME == "master" || LAYOUTNAME == "hy3") {
         // proceed as Hyprland normally would (see CCompositor::swapActiveWorkspaces)
         PWORKSPACEA->m_monitor = PMON2;
-        PWORKSPACEA->moveToMonitor(PMON2->m_id);
+        PWORKSPACEA->m_events.monitorChange.emit();
 
         for (auto& w :g_pCompositor->m_windows) {
             if (w->m_workspace == PWORKSPACEA) {
@@ -231,7 +231,7 @@ SDispatchResult swapActiveWorkspaces(std::string args) {
         }
 
         PWORKSPACEB->m_monitor = PMON1;
-        PWORKSPACEB->moveToMonitor(PMON1->m_id);
+        PWORKSPACEB->m_events.monitorChange.emit();
 
         for (auto& w :g_pCompositor->m_windows) {
             if (w->m_workspace == PWORKSPACEB) {

